scipy/stats/_sensitivity_analysis.py

```diff
@@ -331,13 +338,13 @@ def sobol_indices(
             A method providing confidence intervals on the indices.
             See `scipy.stats.bootstrap` for more details.
 
-            The bootstrapping is done on both first and total order indices
+            The bootstrapping is done on both first and total order indices,
             and they are available in `BootstrapSobolResult` as attributes
             ``first_order`` and ``total_order``.
 
     Notes
     -----
-    Sobol' method [1]_, [2]_ is a variance-based Sensitivity Analysis which
+    The Sobol' method [1]_, [2]_ is a variance-based sensitivity analysis which
```

Comment:
Looks like a style change as well and I would prefer to keep the capitalization.